src/System.IO.Pipes.AccessControl/System.IO.Pipes.AccessControl.sln

```diff
@@ -15,6 +15,9 @@ EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "System.IO.Pipes.AccessControl", "ref\System.IO.Pipes.AccessControl.csproj", "{994DCE47-4CF6-479D-AB1D-4EA6A2809C34}"
 EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "tests", "tests", "{1A2F9F4A-A032-433E-B914-ADD5992BB178}"
+	ProjectSection(SolutionItems) = preProject
+		C:\Users\carlo\.nuget\packages\microsoft.dotnet.codeanalysis\5.0.0-beta.19554.3\content\PinvokeAnalyzer_Win32Apis.txt = C:\Users\carlo\.nuget\packages\microsoft.dotnet.codeanalysis\5.0.0-beta.19554.3\content\PinvokeAnalyzer_Win32Apis.txt
```

Comment:
Need to remove this. It got auto added when I was working on this in my home computer, not entirely sure why.